ibis/backends/pandas/execution/window.py

```diff
@@ -501,17 +501,24 @@ def execute_series_group_by_last_value(op, data, aggcontext=None, **kwargs):
 
 @execute_node.register(ops.MinRank, (pd.Series, SeriesGroupBy))
 def execute_series_min_rank(op, data, **kwargs):
-    # TODO(phillipc): Handle ORDER BY
     return data.rank(method='min', ascending=True).astype('int64') - 1
 
 
 @execute_node.register(ops.DenseRank, (pd.Series, SeriesGroupBy))
 def execute_series_dense_rank(op, data, **kwargs):
-    # TODO(phillipc): Handle ORDER BY
     return data.rank(method='dense', ascending=True).astype('int64') - 1
 
 
-@execute_node.register(ops.PercentRank, (pd.Series, SeriesGroupBy))
+@execute_node.register(ops.PercentRank, pd.Series)
 def execute_series_percent_rank(op, data, **kwargs):
-    # TODO(phillipc): Handle ORDER BY
+    return (data.rank(method='min') - 1) / (len(data) - 1)
```

Comment:
add comments that these are 1 based

Reply:
@jreback could you provide more details about your comment?

Reply:
@jreback, I think I agree with @xmnlab here. In pandas, `df.rank()` for something like `[20, 10, 30]` will return `[2, 1, 3]`. The code is doing `df.rank() - 1`, and then normalizing, so the value should be a percentage between 0 - 1 (as the name `percent_rank` shows). Or do you mean adding a comment to say that the pandas `df.rank()` is 1 based?

Maybe in a follow up would be nice to have docstrings for all them, but this looks ok to me for now, doesn't seem that any of those functions have docstrings anyway.

What I don't understand is why aren't we using the `df.rank(pct=True)` parameter, which seems to be implementing this directly? Am I missing something?

Reply:
thanks for the feedback @datapythonista . Not sure if I am following your question. 
but the point is that the pandas `PercentRank` `df.rank(pct=True)` actually works as `SQL CumeDist`.
So this PR aims to apply the SQL behavior here. As you can see the implementation here for `CumeDist` uses `df.rank(pct=True)` approach.

Reply:
Are `(data.rank(method='min') - 1) / (len(data) - 1)` and `data.rank(method='min', pct=True)` returning different things?

I've seen that `CumeDist` is using `pct=True`, but it's not obvious to me the difference among them if there is one. Besides anwering the question, can you add a comment in the code to explain how this is different from `CumeDist`? This is a bit confusing.

Reply:
Sure I am going to add a comment there.

Just for the record, AFAIK, `col.rank(method='min', pct=True)` would be equivalent to `col.rank(method='min')  / len(col) `

Reply:
@datapythonista instead of adding the docstring here, I just updated the docstring of `PercentRank` and `CumeDist` operation classes. Let me know if it would be enough. I can also add it to the function, but we had a discussion some time ago about adding docstring to these functions (more related to arguments, but I think it also applies to the description), and in that time it was discouraged. Maybe now with the idea of splitting the backends makes more sense. also maybe it could use some approach like the one used by matplotlib (to avoid duplication of docstrings and difficult to handle their changes).

let me know if is there anything else I should do here. thank you so much for the review!

Reply:
i guess this is fine. I know that rank returns 1 based indexing, so pls add a simple comment that indicates this (e.g. explains why you are subtracting 1)